slides/08-Platform-Hub-New/1020-LetsGetOnboarded.md

```diff
@@ -0,0 +1,10 @@
+## Let's get onboarded!
+
+Please follow these step to onboard yourself to the Platform Hub:
+
+ 1. Go to https://access-acp.digital.homeoffice.gov.uk to download and run `ACP NotProd` vpn profile.
```

Comment:
So setting up vpn profiles should not be difficult for developer and since there's a myriad of ways to setup client vpn agents depending on the operating system it makes more sense to keep this open.

Reply:
Agreed; but saying "run" the profile is misleading (as it implies they can just execute the file they download.